oat-sa/tao-i18n-tools
# Feature/ADF-1417/Upgrade to Node 18

### Related to: [ADF-1417](https://oat-sa.atlassian.net/browse/ADF-1417) and [ADF-1419](https://oat-sa.atlassian.net/browse/ADF-1419)

### Summary

Set Node 18 for the CI workflow.

Update the devDependencies accordingly.

### Details

To properly support Node 18, the dev dependencies had to be updated and aligned.

**Note:** The lock file was generated with `npm@6` so it would also work seamlessly with Node 14.

It also removes vulnerabilities 🎉 
The last one would be removed if we can upgrade Svelte
```
audited 427 packages in 1.989s

found 1 moderate severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

### How to test
- check the report on GitHub
- make sure to have Node 18 installed (you can use [nvm.sh](https://github.com/nvm-sh/nvm) for managing multiple versions)
- install the package: `npm i`
- run the unit tests: `npm test`, `npm run test:cov`
- test the linter: `npm run lint`
